clr-k8s-examples/create_stack.sh

```diff
@@ -63,6 +74,12 @@ function cluster_init() {
 	if [ "$(kubectl get nodes | wc -l)" -eq 2 ]; then
 		kubectl taint nodes --all node-role.kubernetes.io/master-
 	fi
+
+	if [[ -n "${HIGH_POD_COUNT}" ]]; then
+		# increase limits in kubelet
+		sudo sed -i 's/^maxPods\:.*/maxPods\: 5000/' /var/lib/kubelet/config.yaml
```

Comment:
i was expecting these two would be updated in kubeadm.yaml, in kubelet config section: https://github.com/clearlinux/cloud-native-setup/blob/044146703f35342bcf58025624128a2eb635ee29/clr-k8s-examples/kubeadm.yaml#L4-L13

Current version of PR only updates master node